bazel/protobuf.bzl

```diff
@@ -21,10 +21,21 @@ def _path_ignoring_repository(f):
         # before "external/workspace", so we need to add the starting index of "external/workspace"
         return f.path[f.path.find(f.owner.workspace_root) + len(f.owner.workspace_root) + 1:]
 
+def _common_suffix(a, b):
```

Comment:
If you wanted to be extra fancy you could just have a common prefix util and use it for both the prefix/suffix cases (with reversed inputs/outputs). But, what you have is fine as well.